tests/unit/HelperTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+define( 'ABSPATH', true );
+
+class HelperTest extends \Codeception\Test\Unit {
+
+	/**
+	 * @var \UnitTester
+	 */
+	protected $tester;
+
+	protected function _before() {
+
+		require_once( 'woocommerce/class-sv-wc-helper.php' );
+	}
+
+	protected function _after() {
+
+	}
+
+	/**
+	 * Tests \SkyVerge\WooCommerce\PluginFramework\v5_10_8\SV_WC_Helper::get_placeholder_list()
+	 *
+	 * @param array $array input country code
+	 * @param string $expected expected return value
+	 *
+	 * @dataProvider provider_can_get_placeholder_list
+	 */
+	public function test_can_get_placeholder_list( array $array, $expected, $placeholder ) {
+
+		$result = \SkyVerge\WooCommerce\PluginFramework\v5_10_8\SV_WC_Helper::get_placeholder_list( $array, $placeholder );
+		$this->assertEquals( $expected, $result );
```

Comment:
You could omit the variable assignment here.

```suggestion
		$this->assertEquals(
			$expected,
			\SkyVerge\WooCommerce\PluginFramework\v5_10_8\SV_WC_Helper::get_placeholder_list( $array, $placeholder )
		);
```